conf/swisnap-init.sh

```diff
@@ -8,34 +8,25 @@ TMP_FILE="${SWISNAP_HOME}/etc/config.yaml.tmp"
 
 # APPOPTICS_TOKEN is required
 if [ -n "${APPOPTICS_TOKEN}" ]; then
```

Comment:
we should also add SOLARWINDS_TOKEN support next to APPOPTICS_TOKEN in near future

Reply:
it must be alligned with AppOptics onboarding